src/Symfony/Component/Config/ConfigCache.php

```diff
@@ -85,8 +85,27 @@ public function isFresh()
             return false;
         }
 
+        $e = null;
+        $meta = false;
         $time = filemtime($this->file);
-        $meta = unserialize(file_get_contents($metadata));
+        $unserializeCallbackHandler = ini_set('unserialize_callback_func', '');
+        $signalingException = new \UnexpectedValueException();
+        set_error_handler(function () use ($signalingException) { throw $signalingException; });
```

Comment:
what about other errors happening there due to bugs in the ``unserialize`` implementation of Serializable resources ? They would be completely swallowed currently. Shouldn't we try to detect only the case of undefined classes ?